elastic/kibana
# [Infra UI] Asset detail view telemetry

Closes #156698

## Summary

This PR adds a new custom event to track asset details page views. The event will have a parameter `integrations` which will show if `nginx` or `kubernetes` integrations are enabled ( so extra charts will be displayed in those cases) and the same parameters as the existing flyout event.

## Testing 

- Open hosts view flyout:
   - The `Asset Details Flyout Viewed` Event type should be tracked (same as before)
   - Event properties in this example (same as before)
   ```
   "properties":{
       "componentName":"infraAssetDetailsFlyout",
       "assetType":"host",
       "tabId":"overview"
     }
   ```
   ![Image](https://github.com/elastic/kibana/assets/14139027/7ab85302-1aed-487c-860c-b0a62ef06a70)

- Open asset details page for a host:
   - The `Asset Details Pade Viewed` Event type should be tracked
   - The event properties should show the current enabled integrations ('nginx', 'kubernetes') if there are no integrations enabled the integrations should not be visible in the event so in this example:
   ```
    "properties": {
       "componentName": "infraAssetDetailsPage",
       "assetType": "host",
       "tabId": "overview",
       "integrations": [
           "nginx"
       ]
   }
   ```
    ![Image](https://github.com/elastic/kibana/assets/14139027/bc657298-cb14-4ac2-ba46-ac42e3bd3e37)
